client/payment-details/summary/missing-order-notice/index.tsx

```diff
@@ -0,0 +1,145 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { Button, RadioControl } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import Loadable from 'wcpay/components/loadable';
+import { useState } from '@wordpress/element';
+import ConfirmationModal from 'wcpay/components/confirmation-modal';
+import interpolateComponents from '@automattic/interpolate-components';
+
+/**
+ * Internal dependencies.
+ */
+
+import './style.scss';
+import CardNotice from 'wcpay/components/card-notice';
+
+interface MissingOrderNoticeProps {
+	isLoading: boolean;
+	formattedAmount: string;
```

Comment:
Non blocking: I'd suggest a different name for this prop since I found `formattedAmount` a bit generic. Maybe `amountToBeRefunded`?

Reply:
We have the amount available from `charge.amount` but we pass it through a method `formatAmount` to generate display value, thus the naming. 

Same value is then used to display the amount of order in the UI and in the modal before refunding, thus putting `refunded` in naming would be weird since it's used outside the "refund".

Reply:
Thanks for explanation. For clarity's sake,  I was referring to the name of the `formattedAmount` property of the `MissingOrderNoticeProps` component, not to the name of the variable.